leaf-ui/rappid-extensions/LinkInspector.js

```diff
@@ -170,11 +170,16 @@ var LinkInspector = Backbone.View.extend({
             
         } else {
             //Verify if it is possible to remove the NB tag from source and target
-            if (!this.hasNBLink(source, this.cell) && this.hasNBTag(source)){
-                source.attr(".funcvalue/text", "");
+            if (!this.hasNBLink(source, this.cell) && !this.hasNBTag(source)){
```

Comment:
Lets talk about how to make sure these changes don't show up in your pull request since they should already be merged into backbone-refactor